fdbclient/CommitProxyInterface.h

```diff
@@ -293,9 +293,13 @@ struct GetKeyServerLocationsReply {
 	// if any storage servers in results have a TSS pair, that mapping is in here
 	std::vector<std::pair<UID, StorageServerInterface>> resultsTssMapping;
 
+	// maps storage server interfaces (captured in "results") to the tags of
```

Comment:
document why we need logic to translate between ssi and tag

Reply:
Instead of adding this, wouldn't it be much easier to change the `getKeyServersLocations` response to also include the tag?

Reply:
> Instead of adding this, wouldn't it be much easier to change the `getKeyServersLocations` response to also include the tag?

You mean changing "results" in GetKeyServerLocationsReply from std::vector<std::pair<KeyRangeRef, vector<StorageServerInterface>>> to std::vector<std::pair<KeyRangeRef, vector<std::pair<StorageServerInterface, Tag>>>> ?

Reply:
Or maybe `Tag` could just be a member of `StorageServerInterface`? The tag should never change during the lifetime of that interface

Reply:
I like this idea.